src/functions.jl

```diff
@@ -345,19 +370,18 @@ for T in (:Taylor1, :TaylorN)
                 return nothing
             end
             zero!(c, k)
-            if $T == Taylor1
-                @inbounds c[k] = k * a[k] * c[0]
-            else
-                @inbounds mul!(c[k], k * a[k], c[0])
-            end
-            @inbounds for i = 1:k-1
+            @inbounds for i = 0:k-1
                 if $T == Taylor1
                     c[k] += (k-i) * a[k-i] * c[i]
                 else
-                    mul!(c[k], (k-i) * a[k-i], c[i])
+                    mul!(c[k], a[k-i], c[i], scalar=k-i)
                 end
             end
-            @inbounds c[k] = c[k] / k
+            if $T == Taylor1
+                @inbounds div!(c, c, k, k)
+            else
+                @inbounds div!(c[k], c[k], k)
+            end
```

Comment:
Same comment as above; in this case, I think the second `if`s can be included in the first.

Reply:
Done.